### ISSUE TITLE
Fresh install error: could not translate host name "postgres" to address: Temporary failure in name resolution

### ISSUE THREAD

**Issue by ssjunior** [getsentry/sentry#40194]
### Environment

self-hosted (https://develop.sentry.dev/self-hosted/)

### Version

22.11.0.dev0

### Steps to Reproduce

Do a fresh install. 

Wait for some time doing nothing else...

### Expected Result

No errors reported.

### Actual Result

Several errors are reported, mainly about dns name resolution. I did a fresh install several times, always get the same issue. The server hosting docker container is resolving dns fine, does not have any name resolution issues.

![Screen Shot 2022-10-18 at 13 48 18](https://user-images.githubusercontent.com/5460671/196493491-fa6e91f2-13c6-44ed-a335-ed28e014b504.jpg)

**Comment by getsentry-release** [comment]
Routing to @getsentry/open-source for [triage](https://develop.sentry.dev/processing-tickets/#3-triage). ⏲️

**Comment by hubertdeng123** [comment]
Hi there, can you paste the logs in your `sentry-self-hosted-postgres-1` container?

**Comment by ssjunior** [comment]
Hi @hubertdeng123. I took a look at sentry-self-hosted-postgres-1 and did not find anything that could be related to this issue.

But, when I was trying to login into server, the server was very sluggish and slow to respond. Investigating I saw a very high cpu/memory consumption and swap memory being used.

After that I changed the memory to 16G and everything is running fine until now. For me it's very strange that this fresh install consume 10G of memory without any activity. It's a fresh install. I do have another old install that uses onlye 2G of memory.

![Screen Shot 2022-10-18 at 17 57 18](https://user-images.githubusercontent.com/5460671/196543488-82c1747a-af97-4015-af72-83c18543f20a.jpg)

This high memory/cpu usage and swap could explain the problem with name resolution. I will install it again, this time with 16G instead of 8G and see if this issue happens again.

Maybe something new was introduced to raise the minimum requirement? 8G defintely is not enough.

**Comment by ssjunior** [comment]
Returning the memory to 8G the issue happens again...

![Screen Shot 2022-10-18 at 18 30 30](https://user-images.githubusercontent.com/5460671/196548936-4711a592-b2ce-4d44-986f-5fb11ba01b90.jpg)


![Screen Shot 2022-10-18 at 18 32 38](https://user-images.githubusercontent.com/5460671/196549042-7a1abb6c-14b1-431f-a158-55eb81ff5f41.jpg)

**Comment by hubertdeng123** [comment]
Gotcha, so what you're saying is that upgrading your version of self-hosted caused an increase in CPU/memory usage that ultimately lead to these errors? My setup is running using ~4 GB of RAM 🤔. 

Just to confirm, this is a fresh install?

**Comment by ssjunior** [comment]
@hubertdeng123 yes, this is a fresh following instructions for that.

I did install, did not add any project and as you cann see in the screenshots above, the memory consumption is very high. I left this vm running after my last comment without adding any projects and this is an actual screenshot of resources used by this install.

![Screen Shot 2022-10-20 at 15 09 52](https://user-images.githubusercontent.com/5460671/197026037-d18ad240-543e-4ee1-8815-529b35ffcde1.jpg)

If I set the memory to 8G the error starts happening again, probably because of CPU/SWAP issues...
